### PR TITLE
Adding test for multibox setup

### PR DESCRIPTION
**Pull request by arcturus**

### CODE REVIEW

File: tests/registration-server-simulator/http-server.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+// Fake registration server:
+// To avoid the new implementation of multiboxes
+// we will be returning always one box, without tunnel
+
+var registrationServerSimulator = express();
+registrationServerSimulator.use(cors());
+
+registrationServerSimulator.get('/ping', (req, res) => {
+  res.setHeader('Content-Type', 'application/json');
+  res.send(JSON.stringify(
```
Comment (JohanLorenzo) [inline tests/registration-server-simulator/http-server.js]: Nit: You could use [res.json()](http://expressjs.com/en/api.html#res.json)

File: tests/registration-server-simulator/http-server.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+var express = require('express');
+var cors = require('cors');
+var path = require('path');
```
Comment (JohanLorenzo) [inline tests/registration-server-simulator/http-server.js]: Nit: Unused import

File: tests/registration-server-simulator/http-server.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+registrationServerSimulator.use(cors());
+
+registrationServerSimulator.get('/ping', (req, res) => {
+  res.setHeader('Content-Type', 'application/json');
+  res.send(JSON.stringify(
+    [{"public_ip":"217.111.161.212","local_ip":"127.0.0.1","tunnel_url":"null","timestamp":Math.floor(Date.now()/1000)}]
+  ));
+});
+
+registrationServerSimulator.listen(4455);
```
Comment (JohanLorenzo) [inline tests/registration-server-simulator/http-server.js]: Not using the default port is a good idea. I should change the one on foxbox simulator too :smiley:

File: gulpfile.js [hunk 4]
```diff
@@ -237,13 +238,17 @@ gulp.task('clean', function(cb) {
   ], cb);
 });
 
-gulp.task('start-foxbox-simulator', function() {
+gulp.task('start-simulators', function() {
   foxboxSimulator = gls.new('tests/foxbox-simulator/http-server.js');
   foxboxSimulator.start();
+
+  registrationServerSimulator = gls.new('tests/registration-server-simulator/http-server.js');
```
Comment (JohanLorenzo) [inline gulpfile.js]: Following up on the live reload issue: I wonder if we shouldn't turn off live reload https://www.npmjs.com/package/gulp-live-server#glsargs-options-livereload

**Comment by JohanLorenzo** [comment]
Looks good to me, modulo some small nits. Regarding live-reload, I believe we can turn it off for both servers.

**Comment by arcturus** [comment]
@JohanLorenzo added your suggestions. Disabled livereload for both servers.

@gmarty do you want to take a look?

**Pull request merged**